source/software/debuggers/index.rst

```diff
@@ -0,0 +1,281 @@
+
+        Once the client is connected, ssh to gaea and start an interactive session to run your code. 
+
+        .. code-block:: shell
+            $ salloc --x11 --clusters=c# --nodes=1 -t1:00:00
+            $ module unload darshan-runtime/3.4.0
+            $ module load forge
+            $ ddt --connect srun -n{number of nodes} ./executable 
+
+
+        .. note::
+            ``ddt --connect`` command must precede the srun command, to open a connection with your (now connected) client on your computer.
+
+        Submit the job.
+
+        When the job begins to run, The  **Reverse Connect Request** prompt displays. 
+
+        1. Click Accept.
+
+        2. Choose any appropriate options, and click Run.
+
+        The debugging interface opens. 
+
+        - Delayed command responses (ex: "ls", "mv") from tfe02 -
+          bastion-hera.princeton
+        - Downloading data to Jet front-end nodes (fe1) using curl
+          hangs/timesout
+
+    .. tab-item:: Hera or Jet
+
+        .. note::
+            Since DDT is GUI debugger, interactions over a wide area network can be extremely slow. You may want to consider using a Remote Desktop which in our environment is X2GO
+
+        For debugging you will need interactive access to the desired set of compute nodes using salloc with the desired set of resources:
+
+        .. code-block:: shell
+            $ salloc --x11=first -N 2 --ntasks=4 -A <project> -t 300 -q batch
+
+        At this point you are on a compute node.
+
+        **Load the desired modules**
+
+        .. code-block:: shell
+            $ module load intel impi forge
+
+        **Launch the application with the debugger**
+
+        .. code-block:: shell
+            % ddt srun -n 4 ./hello_mpi_c-intel-impi-debug
+
+        This will open GUI in which you can do your debugging. Please note that by default it seems to save your current state (breakpoints, etc. are saved for your next debugging session). 
+
+
+
+Launch Jobs Directly from DDT
+-----------------------------
+
+After loading all of the necessary modules, launch ddt GUI from a login node on preferred system. 
+
+.. code-block:: shell
+    $ ddt &
+
+
+.. note::
+    If using on Gaea, from the Remote Launch dropdown, click on the gaea configuration you had set up so your local
+    client connects to Gaea.
+
+
+Once the main window opnes, click on **Run** which should open up a window that looks something like this:
+
+.. figure:: /images/RUNmenu.png
+
+**Application**
+^^^^^^^^^^^^^^
+  
+This lets you specify the path to your application that you want to run.
+
+Click on the 'Details' button on the left to expand this section. Once expanded, you will be able
+to enter:
+
+**Application**: the path to the application
+
+**Arguments**: the args to pass to the application
+
+**stdin file**: any input file needed for the application
+
+**Working Directory**: the working directory from which the job should be started.
+
+**MPI**
```

Comment:
You don't need both Bold and the section heading.  Please remove one or the other.

Note, I think section heading need at least 4 underline characters.  It is OK for the underline to be too long, but it cannot be too short.

Reply:
fixed